ADOPTERS.md

```diff
@@ -18,3 +18,4 @@ This is a list of adopters of using PowerShell in production or in their product
 * [PowerShell Universal Dashboard](https://ironmansoftware.com/powershell-universal-dashboard) is a cross-platform web framework for PowerShell. 
   It provides the ability to create robust, interactive websites, REST APIs, and Electron-based desktop apps with PowerShell script. 
   More information about PowerShell Universal Dashboard is available at the [PowerShell Universal Dashboard Docs](https://docs.universaldashboard.io).
+* [Azure Functions - PowerShell](https://github.com/Azure/azure-functions-powershell-worker) is a service to execute (PowerShell) code or "functions" in the cloud without worrying about compute allocation. In addition, Azure Functions provides client tools such as the [Az.Functions](https://github.com/Azure/azure-powershell/tree/generation/src/Functions) (a cross-platform PowerShell module) to manage function apps and service plans in the cloud. For more information about Functions, please visit [functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview). 
```

Comment:
Please keep the file in ABC sorted format.

Reply:
Yes, please keep new additions in ABC order.